src/components/WebexActivity/ActivityHeader.test.js

```diff
@@ -0,0 +1,57 @@
+import React from 'react';
+import {addDays, addHours, startOfToday, startOfWeek, subDays} from 'date-fns';
+
+import ActivityHeader, {formatMessageDate} from './ActivityHeader';
+
+jest.mock('../hooks/usePerson');
+
+describe('Activity Header component', () => {
+  let today, sunday;
+
+  beforeEach(() => {
+    today = startOfToday(); // Mock date is set to Thu, Aug 1 2019
```

Comment:
Can you put that it starts at 00:00 as well?

Reply:
updated